js_modules/dagit/packages/core/src/app/time/TimezoneSelect.tsx

```diff
@@ -58,7 +58,10 @@ export const TimezoneSelect: React.FC<Props> = ({trigger}) => {
 
   return (
     <SelectWIP<typeof SortedTimezoneItems[0]>
-      popoverProps={{position: 'bottom-left'}}
+      popoverProps={{
+        position: 'bottom-left',
+        modifiers: {offset: {enabled: true, offset: '-12px, 8px'}},
+      }}
```

Comment:
Unrelated to colors, but the positioning of the new Popover bugged me so I changed it.